keycloak/keycloak-admin-ui
# Initial access token list and create

## Motivation

list the initial access tokens in a tab on the clients list see [marvel](https://marvelapp.com/prototype/fcf2450/screen/74745139)

## Verification Steps

1. Go to clients list and see the new "Inital access token" tab
1. Click create on the table toolbar and create a new one
1. See that it's added to the list

## Checklist:


## Additional Notes
